### PR TITLE
docs: new getting started with embedded

### PR DESCRIPTION
**Pull request by avarobinson**
# Pull Request Checklist

- [ ] Did you add new tests and confirm existing tests pass? (`yarn test`)
- [ ] Did you update relevant docs? (docs are found in the `site` folder, and guidelines for updating/adding docs can be found in the [contribution guide](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md))
- [ ] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [ ] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:write`)
- [ ] Did you follow the [contribution guidelines](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md)?

---

## PR-Codex overview
The focus of this PR is to introduce the Alchemy Signer for building embedded accounts with email, passkey, and social auth flows. 

### Detailed summary
- Added a tip to use Alchemy Signer for secure user-friendly Embedded Account
- Updated "Dive deeper" section to "Use Alchemy Signer" with guide link
- Added new section in `alchemy-signer/index.md` for creating account config
- Included images for account config creation in `alchemy-signer/index.md`
- Updated usage instructions for Alchemy Signer in `alchemy-signer/index.md`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: site/signers/alchemy-signer/index.md [hunk 1]
```diff
@@ -22,14 +22,26 @@ head:
 
 The Alchemy Signer is a `SmartAccountSigner` that is powered by Alchemy's Signer Infrastructure. Using the Alchemy Signer, you can get started building embedded accounts with just an Alchemy API key!
 
-## Usage
+## Create an account config
+
+To use the Alchemy Signer, you will need to create a new account config in your [Alchemy Accounts Manager Dashbord](https://dashboard.alchemy.com/accounts/) and connect this to your apps.
```
Comment (avasisht23) [inline site/signers/alchemy-signer/index.md:27]:
```suggestion
To use the Alchemy Signer, you will need to create a new account config in your [Alchemy Accounts Manager Dashboard](https://dashboard.alchemy.com/accounts/) and connect this to your apps.
```

**Review by avasisht23 — APPROVED** [review]
small typo, otw lgtm

**Review by rthomare — APPROVED** [review]

File: site/signers/alchemy-signer/index.md [hunk 2]
```diff
@@ -42,6 +54,10 @@ Once you have initialized your signer, you can now enable your users to create a
 
 Once your signer is authenticated with a user, you can use it to sign User Operations by creating a `SmartContractAccount` and passing the signer to it.
 
+::: tip
```
Comment (moldy530) [inline site/signers/alchemy-signer/index.md:57]: let's move this to the top of the doc

File: site/getting-started/setup.md [hunk 3]
```diff
@@ -95,6 +95,10 @@ To read or write any data to a blockchain, you will need an Alchemy API Key and
 
 ## 3. Query your Smart Account Address
 
+::: tip
+Don't use the example below with a hardcoded private key in production. Follow the [Alchemy Signer guide](/signers/alchemy-signer) to build a secure user friendly Embedded Account!
```
Comment (rthomare) [inline site/getting-started/setup.md:99]: Don't use hardcoded private keys in production, like the example below. Instead, follow this [Alchemy Signer ...

**Review by rthomare — APPROVED** [review]
One nit to consider before pushing.

File: site/signers/alchemy-signer/index.md [hunk 4]
```diff
@@ -22,14 +22,34 @@ head:
 
 The Alchemy Signer is a `SmartAccountSigner` that is powered by Alchemy's Signer Infrastructure. Using the Alchemy Signer, you can get started building embedded accounts with just an Alchemy API key!
 
-## Usage
+Learn how to set up your account config, create an `AlchemySigner`, log users in with email auth, and create a embedded account with our signer to enable email, passkeys (i.e. biometrics), and soon social auth flows!
+
+::: tip
```
Comment (rthomare) [inline site/signers/alchemy-signer/index.md:27]: move this up in the doc.

**Review by rthomare — APPROVED** [review]

**Pull request merged**